requester/requester.go

```diff
@@ -222,7 +232,15 @@ func (b *Work) runWorker(n int) {
 	} else {
 		tr.TLSNextProto = make(map[string]func(string, *tls.Conn) http.RoundTripper)
 	}
-	client := &http.Client{Transport: tr, Timeout: time.Duration(b.Timeout) * time.Second}
+
+	var checkRedirect func(req *http.Request, via []*http.Request) error
+	if b.DisableRedirects {
+		checkRedirect = func(req *http.Request, via []*http.Request) error {
+			return errRedirectNotAllowed
```

Comment:
return http.ErrUseLastResponse instead. http.Client knows what http.ErrUseLastResponse is.